sio/workers/executors.py

```diff
@@ -670,6 +670,7 @@ def _execute(self, command, **kwargs):
                     "Unrecognized Sio2Jail output, stderr: %s" % full_result
                 )
 
+
```

Comment:
a 1-line break is enough, I think that a 2-line one symbolises an end of a function etc.